python/tests/rtl/test_serialize.py

```diff
@@ -99,6 +99,7 @@ class Model(ml.Model):
     # standard types
     id: Optional[int] = None
     name: Optional[str] = None
+    datetime_field: Optional[datetime.datetime] = None
```

Comment:
adding this `datetime_field` everywhere in the tests identically reproduced the issue in #720